azafea/event_processors/endless/activation/v1/handler.py

```diff
@@ -65,6 +66,28 @@ def validate_country(self, key: str, country: Optional[str]) -> Optional[str]:
 
         return country
 
+    @validates('latitude')
+    def validate_latitude(self, key: str, latitude: Optional[int]) -> Optional[int]:
+        # Catch both None and the empty string
+        if not latitude:
+            return None
+
+        if math.floor(latitude) + 0.5 != latitude:
+            raise ValueError(f'latitude has wrong format: {latitude}')
```

Comment:
We can be more specific in this error, perhaps something like:

```suggestion
            raise ValueError(f'latitude is not an integer + 0.5: {latitude}')
```

Reply:
or "latitude has not had its precision reduced"?